packages/api/docs/future/paths/authentication.yaml

```diff
@@ -0,0 +1,170 @@
+login:
+  post:
+    summary: Log in to the Permanent API. May require two-factor authentication.
+    security: []
+    operationId: login
+    tags:
+      - authentication
+    requestBody:
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              required: [email, password, applicationId]
+              email:
+                type: string
+              password:
+                type: string
+              applicationId:
```

Comment:
Note that this API will eventually include endpoints for managing applications, but these are not included in this PR because we need to do some significant research and design work on those and I want to get examples of OpenAPI documentation into the repo.